src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: How to use AMP in Next.js (and why)
+authors:
+  - houssein
+subhead: |
+  Add AMP to any of your Next.js pages to guarantee fast page loads
+date: 2019-10-17
+draft: true
+---
+
+[AMP](https://amp.dev) (Accelerated Mobile Pages) is a web component framework that guarantees
+that your pages will load fast. [Next.js][intro] has built-in support for AMP.
+
+## What will you learn?
+
+This guide first briefly describes [how AMP guarantees fast page loads](#overview), then lets you
+try out [the two ways that you can use AMP in a Next.js app](#strategies), then helps you [decide
+which approach is best for you](#guidance).
+
+The intended audience for this guide is a web developer who has decided to use Next.js but is
+unsure of whether to support AMP. 
```

Comment:
Aside from helping readers evaluate whether this page is right for them, being explicit about the audience also helps us (the web.dev maintainers) remember the original purpose of this document.

We might want to go a step further and define our audience as a web developer who has decided to use Next.js and **considers load performance a high priority**. Because load performance optimization is the unifying theme behind all of our Next.js content.

Reply:
Love it, definitely helps to be more clear 👍 

I'm okay with both!